project_4/styles.css

```
@@ -0,0 +1,667 @@
+    max-width: 100%;
+    padding: 0 20px;
+    text-shadow: 7px 7px 10px var(--color-brown);
+    text-transform: uppercase;
+    width: 300px;
+    z-index: 1;
+}
+
+/* REGISTER BUTTON */
+.home-page-register {
+    background-color: var(--color-blue);
+    border: 5px dashed var(--color-offwhite);
+    color: var(--color-brown);
+    display: inline-block;
+    font-family: var(--font-header);
+    font-size: 20px;
+    font-weight: 700;
+    margin: 30px 30px;
+    padding: 30px 50px;
+    text-align: right;
+    text-decoration: none;
+    text-transform: uppercase;
+    transition: transform .2s;
+}
+
+.home-page-register:hover {
+    background-color: var(--color-brown);
+    color: var(--color-offwhite);
+    transform: scale(1.1);
+}
+
+/* ============= */
+
+h1,
+h3,
+h2 {
+    font-family: var(--font-header)
+}
+
+h2 {
+    color: var(--color-blue);
+    font-size: 50px;
+    line-height: 1;
+    text-transform: uppercase;
+}
+
+/* ================
+MISSION 
+==================*/
+
+h2,
+h3,
+.quotes {
+    text-align: center;
+}
+
+.mission-text {
+    font-family: var(--font-paragraph);
+    line-height: 1.5;
+    text-align: center;
+}
+
+.section-header {
+    background-color: var(--color-brown);
+    font-size: 60px;
+    margin-top: 100px;
+    padding: 100px 0;
+    text-transform: uppercase;
+    width: 100%;
+}
+
+h3 {
+    font-size: 50px;
+    line-height: 1;
+    /* margin-top: 80px; */
+}
+
+.quotes {
+    background-color: var(--color-green);
+    color: white;
+    font-size: 20px;
+    margin-top: 70px;
+    padding: 30px;
+}
+
+.quotes h3 {
+    color: rgb(158, 158, 158);
+}
+
+/* next button on quotes */
+.button-reset {
+    background-color: var(--color-brown);
+    border: 2px dashed var(--color-offwhite);
+    border-radius: 50%;
+    color: var(--color-offwhite);
+    font-family: var(--font-paragraph);
+    font-size: 15px;
+    text-transform: uppercase;
+    transition: transform 1s;
+    height: 90px;
+    width: 90px;
+}
+
+.button-reset:hover {
+    background-color: var(--color-blue);
+    color: var(--color-brown);
+    transform: rotate(360deg);
+}
+
+.left-column,
+.right-column {
+    margin: auto;
+    width: 80%;
+}
+
+
+/* ================
+SPEAKER SECTION
+=================== */
+
+.speakers {
+    margin: auto;
+    width: 80%;
+}
+
+.speakers img {
+    border: 8px dashed var(--color-brown);
+    border-radius: 50%;
+    margin-bottom: 10px;
+}
+
+.speaker-info {
+    list-style-type: none;
+    margin-bottom: 70px;
+    padding: 0;
+    text-align: center;
+
+}
+
+.speaker-name,
+.speaker-title {
+    font-family: var(--font-header);
+}
+
+.speaker-name {
+    font-weight: 700;
+    text-transform: uppercase;
+}
+
+.speaker-title {
+    margin-bottom: 10px;
+}
+
+.speaker-name,
+.speaker-title {
+    font-size: 18px;
+}
+
+.speaker-paragraph {
+    font-family: var(--font-paragraph);
+}
+
+/* hover effects for speakers */
+.speaker-info img:hover {
+    border: 8px dashed var(--color-blue);
```

You can simplify this by only updating the bit that changes on hover:
```suggestion
    border-color: var(--color-blue);
```